airbyte-integrations/connectors/source-trello/source_trello/schemas/boards.json

```diff
@@ -61,7 +61,7 @@
       "type": ["null", "string"]
     },
     "ixUpdate": {
-      "type": ["null", "integer"]
```

Comment:
From Trello documentation:
https://developer.atlassian.com/cloud/trello/rest/api-group-boards/#api-boards-id-get

<img width="767" alt="image" src="https://user-images.githubusercontent.com/5154322/215050520-e5a1b0e1-4672-4d7a-9e6c-0393f617c644.png">

<img width="319" alt="image" src="https://user-images.githubusercontent.com/5154322/215050622-b664df8f-41bd-4419-8f8f-8cb9f2020058.png">

Maybe you can show us an example with string from your dataset.

Reply:
@sajarin 
This endpoint is used in the stream: https://developer.atlassian.com/cloud/trello/rest/api-group-members/#api-members-id-boardstars-idstar-delete
reference: https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-trello/source_trello/source.py#L86
Seems like the Trello doc is also not upto-date.

Here you can see in the screenshot that ixUpdate is a string.

<img width="1132" alt="ixupdated-wrong" src="https://user-images.githubusercontent.com/16688852/215060500-eae256d8-e275-4883-bda5-3c9959cc3d5d.png">